classes/class-woothemes-sensei-certificates.php

```diff
@@ -127,7 +127,7 @@ public function __construct( $file ) {
 		}
 
 		// Generate certificate hash when course is completed.
-		add_action( 'sensei_user_course_end', array( $this, 'generate_certificate_number' ), 10, 2 );
+		add_action( 'sensei_course_status_updated', array( $this, 'generate_certificate_number' ), 9, 4 );
```

Comment:
Shouldn't this be 3?

Reply:
Oof. Guess I need a weekend 😅 

Good catch. Also fixed in abe5833